zoo-css-challenge/style.css

```
@@ -105,33 +114,55 @@ footer {
 
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
 }
 
-.image-container img:first-of-type {
-  vertical-align: bottom;
+.image-container .bear1 {
+  border: 5px solid #890add;
 }
 
+.image-container .bear2 {
+  border: 5px solid #f7df0a;
+}
+
+.image-container .bear3{
+  border: 5px solid #890add;
+}
 .tigers {
-  background-color: #0b3c49;
-  color: #f0e100;
+  background-color: #f1e684;
+  color: #36362b;
 }
 
 .giraffe {
-  background-color: #465775;
+  background-color: #483C46;
   color: #56e39f;
 }
 
 .giraffe h2 {
   color: #ef6f6c;
 }
 
+p:first-of-type{
```

```suggestion
.giraffe p:first-of-type{
```
The task is to make the first paragraph larger in the giraffe section, so we can add `.giraffe` to make sure we only increase the size in the giraffe section

---

Thank you for your advice!